samples/dashboards/demo/accounting/demo.css

```diff
@@ -43,15 +47,15 @@
     color: #aaa;
 }
 
-@media (max-width: 965px) {
+@container dashboard (max-width: 975px) {
```

Comment:
Can we use the width, used in other media queries? We will keep the consistency.

```
/* LARGE */
@media (max-width: 1200px) {

}

/* MEDIUM */
@media (max-width: 992px) {

}

/* SMALL */
@media (max-width: 576px) {

}
```